src/components/Gauge.js

```diff
@@ -0,0 +1,263 @@
+      hoveredArc,
+      pie,
+      radius,
+      standardArc,
+      values
+    });
+  },
+
+  _renderArcs () {
+    const firstSegment = -135 * (Math.PI / 180);
+    const endOfFirstSegment = -90 * (Math.PI / 180);
+    const secondSegment = -90 * (Math.PI / 180);
+    const endOfSecondSegment = -45 * (Math.PI / 180);
+    const thirdSegment = -45 * (Math.PI / 180);
+    const endOfThirdSegment = 0;
+    const fourthSegment = 0;
+    const endOfFourthSegment = 45 * (Math.PI / 180);
+    const fifthSegment = 45 * (Math.PI / 180);
+    const endOfFifthSegment = 90 * (Math.PI / 180);
+    const sixthSegment = 90 * (Math.PI / 180);
+    const endOfSixthSegment = 135 * (Math.PI / 180);
+
+    const segments = [
+      { startAngle: firstSegment, endAngle: endOfFirstSegment, padAngle: 0.02 },
+      { startAngle: secondSegment, endAngle: endOfSecondSegment, padAngle: 0.02 },
+      { startAngle: thirdSegment, endAngle: endOfThirdSegment, padAngle: 0.02 },
+      { startAngle: fourthSegment, endAngle: endOfFourthSegment, padAngle: 0.02 },
+      { startAngle: fifthSegment, endAngle: endOfFifthSegment, padAngle: 0.02 },
+      { startAngle: sixthSegment, endAngle: endOfSixthSegment, padAngle: 0.02 }];
+
+    return segments.map((point, i) => {
+      return (
+        <g
+          key={i}
+        >
+          <path
+            className={'arc-' + this.props.id}
+            d={this.state.standardArc(point)}
+            fill={this.props.colors[i]}
+            opacity={this.props.opacity}
+            ref={'arc-' + this.props.id + i}
+          />
+        </g>
+      );
+    });
+  },
+
+  _renderBaseArc () {
+    return (
+      <g>
+        <path d={this.state.baseArc()} fill={this.props.baseArcColor}></path>
+      </g>
+    );
+  },
+
+  _renderDataPoints () {
+    const dataPoints = this.props.dataPoints.map(dataPoint => {
+      return dataPoint.value;
+    });
+
+    return dataPoints.map((dataPoint, index) => {
+      const percentOfTotal = dataPoint / this.props.chartTotal;
+      const endAngle = (percentOfTotal * 270) - 135;
+
+      const dataPointArc = d3.svg.arc()
+        .outerRadius(this.state.radius - this.props.activeOffset)
+        .innerRadius(this.state.radius - this.props.arcWidth)
+        .startAngle(endAngle * (Math.PI / 180))
+        .endAngle(endAngle * (Math.PI / 180));
+
+      return (
+        <circle
+          cx='0'
+          cy='0'
+          fill={this.props.dataPointColors[index]}
+          key={index}
+          r={this.props.dataPointRadius}
+          transform={'translate(' + dataPointArc.centroid() + ')'}
+        />
+      );
+    });
+  },
+
+  _renderDataLabel () {
+    const styles = this.styles();
+
+    if (this.props.showDataLabel) {
+      if (this.props.children) {
+        return (
+          <div
+            className='mx-gauge-data'
+            onClick={this._handleClick}
+            style={styles.center}
+          >
+            {this.props.children}
+          </div>
+        );
```

Comment:
Since this is the only if statement in the true branch of the outer if statement, can we just combine them?

```javascript
if (this.props.showDataLabel && this.props.children) {
```